### ISSUE TITLE
[BUG] Documentation should state that monitoring will not work unless memory is increased from default

### ISSUE THREAD

**Issue by adelton** [code-ready/crc#810]
### General information

When attempting to follow https://code-ready.github.io/crc/#starting-monitoring-alerting-telemetry_gsg to enable monitoring, alerting, and telemetry, errors are shown due to insufficient memory. It's OK that by default the VM only has 8 GiB and that by default monitoring is not run, but the requirement for more memory (how much? 16 GB worked for me but maybe it's too much) should be spelled out, together with information how to achieve it.

  * OS: Linux
  * Hypervisor: KVM
  * Did you run `crc setup` before starting it (Yes/No)? Yes

## CRC version
```bash
crc version: 1.1.0+95966a9
OpenShift version: 4.2.2 (embedded in binary)
```
  
## CRC status
```bash
CRC VM:          Running
OpenShift:       Running (v4.2.2)
Disk Usage:      13.64GB of 32.2GB (Inside the CRC VM)
Cache Usage:     9.467GB
Cache Directory: /home/test/.crc/cache
```

## CRC config
Empty output.

## Host Operating System
```bash
NAME="Red Hat Enterprise Linux"
VERSION="8.1 (Ootpa)"
ID="rhel"
ID_LIKE="fedora"
VERSION_ID="8.1"
PLATFORM_ID="platform:el8"
PRETTY_NAME="Red Hat Enterprise Linux 8.1 (Ootpa)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:redhat:enterprise_linux:8.1:GA"
HOME_URL="https://www.redhat.com/"
BUG_REPORT_URL="https://bugzilla.redhat.com/"

REDHAT_BUGZILLA_PRODUCT="Red Hat Enterprise Linux 8"
REDHAT_BUGZILLA_PRODUCT_VERSION=8.1
REDHAT_SUPPORT_PRODUCT="Red Hat Enterprise Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="8.1"
```

### Steps to reproduce

  1. Setup and start CRC with the defaults, `crc start -p pull.secret`
  2. `oc login` as `kubeadmin`
  2. Follow https://code-ready.github.io/crc/#starting-monitoring-alerting-telemetry_gsg to start monitoring, run `oc scale --replicas=1 statefulset --all -n openshift-monitoring; oc scale --replicas=1 deployment --all -n openshift-monitoring`
  3. Check https://console-openshift-console.apps-crc.testing/k8s/all-namespaces/events

### Expected

No errors.

### Actual

Events like

prometheus-adapter-748788db5d-r5xkjNamespaceNSopenshift-monitoring
less than a minute ago
Generated from default-scheduler
0/1 nodes are available: 1 Insufficient memory.

community-operators-76d7d5f759-4f2ncNamespaceNSopenshift-marketplace
a few seconds ago
Generated from default-scheduler
5 times in the last minute
0/1 nodes are available: 1 Insufficient memory.

etcd-quorum-guardNamespaceNSopenshift-machine-config-operator
Nov 14, 3:35 pm
Generated from controllermanager
14 times in the last 7 minutes
0/1 nodes are available: 1 Insufficient memory.

When I then delete the CRC and start it anew with `-m 16000` to give the VM more memory, the problem goes away.

### Logs

You can start crc with `crc start --log-level debug` to collect logs.
Please consider posting this on http://gist.github.com/ and post the link in the issue.

**Comment by gbraad** [comment]
This could be part of the "Requirements', but we have neglected to mention this as it would have been obvious. Seems it is not ;-).

I would wonder what the minimum in that case would be... 12G ?

**Comment by gcharters** [comment]
I think it would be good to put something in the docs, e.g. section 2.1 on this page https://access.redhat.com/documentation/en-us/red_hat_codeready_containers/1.0/html/getting_started_guide/common-tasks_gsg

I've just bumped into this, and it would have been nice if the docs had flagged it wouldn't work before I ran `oc scale --replicas=1 statefulset --all -n openshift-monitoring; oc scale --replicas=1 deployment --all -n openshift-monitoring`

**Comment by robin-owen** [comment]
Fixed by #1056.
